workers/typescript/src/payload-converter.ts

```diff
@@ -0,0 +1,58 @@
+import root, { temporal } from './protos/root';
+import {
+  ProtobufBinaryPayloadConverter,
+  ProtobufJsonPayloadConverter
+} from '@temporalio/common/lib/converter/protobuf-payload-converters';
+import {
+  BinaryPayloadConverter,
+  CompositePayloadConverter,
+  JsonPayloadConverter,
+  PayloadConverterWithEncoding,
+  UndefinedPayloadConverter,
+  ValueError
+} from '@temporalio/common';
+import { decode, encode } from '@temporalio/common/lib/encoding';
+import Payload = temporal.api.common.v1.Payload;
+
+export class PassThroughPayload implements PayloadConverterWithEncoding {
+
+  public toPayload(value: any): Payload | undefined {
+    if (!value || value.metadata === undefined || value.data === undefined) {
+      return undefined;
+    }
```

Comment:
If there's a less stupid way to do this, please tell me. `instanceof` doesn't work. `Payload.verify` might be best, but we're not codgen-ing that right now.